core/src/main/scala/com/mesosphere/usi/core/logic/MesosEventsLogic.scala

```diff
@@ -19,25 +18,38 @@ import scala.collection.JavaConverters._
 private[core] class MesosEventsLogic(mesosCallFactory: MesosCalls, offerMatcher: OfferMatcher = new FCFSOfferMatcher())
     extends ImplicitStrictLogging {
 
+  val podTaskIdStrategy: PodTaskIdStrategy = PodTaskIdStrategy.DefaultStrategy
```

Comment:
This will be settable later on, so we can use different strategies for e.g. legacy metronome IDs, right?

Reply:
Yeah, that's the idea. Not all frameworks will have the same opinion about how task names in a taskGroup related to a taskId.